pkgs/applications/networking/protonmail-bridge/default.nix

```diff
@@ -67,6 +78,13 @@ in stdenv.mkDerivation rec {
       $out/lib/protonmail-bridge
   '';
 
+  postInstall = ''
```

Comment:
We have qt wrappers now so we don't need to roll our own anymore.

Please read
* https://hydra.nixos.org/build/96749518/download/1/nixpkgs/manual.html#sec-language-qt

to see how to adapt the expression to use it.